app/elements/login-popup.html

```diff
@@ -90,7 +97,7 @@
                   label="Username" type="text" value="{{username}}"
                   on-keydown="enterAndTryRegister"></paper-input>
               <paper-input required auto-validate error-message="needs a number!"
-                  label="Student number" type="number" value="{{studentnumber}}"
+                  label="Student number" type="number" value="{{studentnumber}}" min=1000000 max=9999999
```

Comment:
nah, but study numbers are at the rise fast, and may hit the max value at some point in a few years... but like I said, wishful thinking